data/projects/editor_support.md

```diff
@@ -13,6 +13,28 @@ plugins. For some editors like sublime and gedit, running tests is not easy and
 there is no api to do it. In such cases we  would want to use dogtail
 (a GUI tester) or a similar tool to do this.
 
+Other details (for testing): 
+
+For sublime, we'll be using the unittest plugin which is a plugin by itself. 
+For atom, we have `apm test`.
+For VI and emacs, we will most likely use a testing framework such as Vimcasts.
+
+Other details (for maintenance):
+
+We will have to explore the possibility of having a single git repository for 
+atom, sublime-text and vim as it will ultimately be easier to maintain in the
+long run and will have a proper CI, which can be easily managed. Issues will 
+be raised more frequently as a result of this.
```

Comment:
Can't the CI be just copied files and these repos be different. What's the benefit of having the same repo can you elaborate a bit on that point wise?